.github/workflows/delete-v2-review-app.yml

```diff
@@ -4,106 +4,62 @@ on:
   pull_request:
```

Comment:
the workflow in the guidance to review didn't delete anything? terraform-destroy returned 'No changes. No objects need to be destroyed.'